File: sample/java/TestBasic.java
public class TestBasic {
    public static void main(String[] args) {
        System.out.println("Starting application");
        System.out.println("debug: initialization started");

        int x = 42;
        System.out.printf("Value: %d\n", x);

        System.err.println("error: something went wrong");
        System.err.println("DEBUG: detailed error info");

        System.out.print("inline message");

        int result = calculate(10, 20);
        System.out.println("Result: " + result);
    }

    static int calculate(int a, int b) {
        System.out.printf("debug: calculating %d + %d\n", a, b);
        return a + b;
    }
}

File: sample/go/test_basic.go
package main

import (
	"fmt"
	"log"
)

func main() {
	fmt.Println("Starting application")
	fmt.Println("debug: initialization started")

	x := 42
	fmt.Printf("Value: %d\n", x)

	log.Println("error: something went wrong")
	log.Println("DEBUG: detailed error info")

	fmt.Print("inline message")

	result := calculate(10, 20)
	fmt.Println("Result:", result)
}

func calculate(a, b int) int {
	fmt.Printf("debug: calculating %d + %d\n", a, b)
	return a + b
}
